lib/web_ui/lib/src/engine/html/path/path.dart

```diff
@@ -1537,6 +1537,12 @@ class SurfacePath implements ui.Path {
   /// a persistent div.
   ui.Rect? get webOnlyPathAsRect => pathRef.getRect();
 
+  /// Detects if path is a vertical or horizontal line returns LTRB or null.
+  ///
+  /// Used for web optimization of physical shape represented as
+  /// a persistent div.
+  ui.Rect? get webOnlyPathAsLine => pathRef.getLine();
```

Comment:
This doesn't need the `webOnly` prefix as it's in the private `_engine` library.

Reply:
Done.